archetypes/helidon/src/main/archetype/mp/oci/files/server/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,19 @@
+# Application properties. This is the default greeting
+app.greeting=Hello
+
+# Microprofile server properties
+server.host=0.0.0.0
+server.port=8080
+
+# Turn on support for REST.request SimpleTimers for all JAX-RS endpoints
+metrics.rest-request.enabled=true
+
+# OCI Monitoring configuration
+oci.monitoring.compartmentId=<your monitoring compartment id>
```

Comment:
nice to have: validation (via assert or something) that these config values were populated (i.e., doesn't contain "<" or ">")